app/models/journaled/bulk_writer.rb

```diff
@@ -0,0 +1,44 @@
+class Journaled::BulkWriter
+  EVENTS_PER_DELIVERY = 500
```

Comment:
Might eventually want this to be configurable in an initializer like some of the other `Journaled` attributes, but I think leaving it as a constant for now is likely good enough!

Reply:
just checking the kinesis docs: "Each PutRecords request can support up to 500 records. Each record in the request can be as large as 1 MiB, up to a limit of 5 MiB for the entire request, including partition keys. Each shard can support writes up to 1,000 records per second, up to a maximum data write total of 1 MiB per second."

if we're enqueueing a bunch of 500-record bulk deliveries, it seems like we might hit that 1000 record/second limit kind of easily. do we want to try staggering the enqueued jobs, maybe?

Reply:
Ya I think we should at least add the option to have some stagger to the jobs, that's a good call!

Reply:
Pushed a commit for this!

I think this will work for now, but I think eventually I want to refactor it a bit, and def before we upstream this.

I think I can probably refactor such that `enqueue_opts` could also be a callable that you pass in, which I _think_ would allow for us to do something like this at the app level instead of in the framework. But that's a task for later I think!